src/registerAdapters.ts

```diff
@@ -35,7 +36,21 @@ function registerAdapterForServer(serverName: string, server: Server, app: Glee,
     // TODO: Implement AMQP support
   } else if (['ws', 'wss'].includes(protocol)) {
     const configWsAdapter = config?.websocket?.adapter
-    if (!configWsAdapter || configWsAdapter === 'native') {
+    /**
+     * We access the `x-kind` parameter from server object
+     */
+    const kind = server.json('x-kind');
+    /**
+     * If the server is remote we create client adapter otherwise 
+     * we create server adapter. 
+     */
+    if (kind === 'remote') {
```

Comment:
I think we should default to client whenever possible. Glee is a client for multiple protocols. It should be an exception that it behaves as a server IMHO.